docs/ConfigureScalarDL.md

```diff
@@ -2,126 +2,43 @@
 
 This guide explains how to configure Scalar DL custom values in helm charts for Scalar DL deployment.
 
-## Create Kubernetes Secrets
-
-Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. 
-This optional method is highly recommended for handling credentials in the production environment.
-
-You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
-
-```
-# Cosmos DB
-    # username=<Account name>
-    # password=<Cosmos DB account primary master key>
-
-# DynamoDB
-    # username=<AWS_ACCESS_KEY_ID>
-    # password=<AWS_ACCESS_SECRET_KEY>
-
-kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
-```
-
-## Update Configuration Files 
-
-### Cosmos DB
-
-#### Configure schema-loading-custom-values
-
-To create a Scalar DL schema in Cosmos DB, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
-
-```yaml
-schemaLoading:
-  existingSecret: scalardl
-
-  database: cosmos
-  contactPoints: <Cosmos DB account endpoint>
-  cosmosBaseResourceUnit: 400
-```
-
-**Without Kubernetes secrets**
-
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
-
-```yaml
-schemaLoading:
-  database: cosmos
-  contactPoints: <Cosmos DB account endpoint>
-  password: <Cosmos DB account primary master key>
-  cosmosBaseResourceUnit: 400
-```
-
-#### Configure scalardl-custom-values 
-
-To deploy Scalar DL Ledger on Cosmos DB, you need to update the [scalardl-custom-values.yaml](../conf/scalardl-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
 
-```yaml
-ledger:
-  existingSecret: scalardl
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbStorage: cosmos
-```
-**Without Kubernetes secrets**
+## Reference Table
 
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
+You can refer to the table below to get values to create Kubernetes Secrets and update the configuration files for Scalar DL deployment according to the database you chose.
 
-```yaml
-ledger:
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
-```
+| Database  | storageType | contactPoints              | username       | password                                 | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
```

Comment:
It seems that this table does not include Cassandra information. Is there any reason?

Reply:
@kota2and3kan san
Cassandra is not included because it was not tested (fully managed cloud-based Cassandra) and included in the database document (to set up a database for scalar DL) updates. Hence I didn't add it to the document.